examples/prismic/components/Builtins/Blog/Blog.makeswift.ts

```diff
@@ -0,0 +1,16 @@
+import { Slot, Style } from '@makeswift/runtime/controls'
+
+import { runtime } from '@/lib/makeswift/runtime'
+
+import Blog from './Blog'
+
+export const BLOG_EMBEDDED_COMPONENT_ID = 'blog-component' //unique id for the registered component
```

Comment:
See previous comment about not using the ID for both the type and the snapshot ID. Also, I think that we can keep this type encapsulated here by exporting an RSC that fetches the snapshot:

```tsx
export async function MakeswiftBlog({ id, label = "Blog Post" }: { id: string; label?: string }) {
  const snapshot = await client.getComponentSnapshot(id)

  return <MakeswiftComponent type={TYPE} label={label} snapshot={snapshot} />
}
```